card_db_src/sets_db.json

```diff
@@ -81,7 +81,7 @@
             "upgrade"
         ],
         "has_extras": false,
-        "image": "cornucopia_set.png",
+        "image": "cornucopia_guilds_set.png",
```

Comment:
@sumpfork I think the Upgrade Set should get the upgraded icon too. 
(I didn't rebuild the `sets_db.json.gz` because I'm a limited connection but I think the build scripts do that)
Otherwise looks good. Thanks.

Reply:
Actually, I got `doit update_languages` to work and uploaded that `sets_db.json.gz` so I think it's all good now.